flags.go

```diff
@@ -3,6 +3,11 @@ package main
 import "github.com/codegangsta/cli"
 
 var (
+	flStore = cli.StringFlag{
+		Name:  "store",
+		Value: "/var/lib/docker/swarm/store",
```

Comment:
I think we should change the so you don't have to `sudo` or change `--store` every time.

`$HOME/.swarm/store` ?